src/groups/mqb/mqbnet/mqbnet_initialconnectioncontext.h

```diff
@@ -131,19 +134,22 @@ class InitialConnectionContext {
     InitialConnectionContext&
     setChannel(const bsl::shared_ptr<bmqio::Channel>& value);
     InitialConnectionContext&
-    setInitialConnectionCompleteCb(const InitialConnectionCompleteCb& value);
+    setCompleteCb(const InitialConnectionCompleteCb& value);
     InitialConnectionContext&
     setNegotiationContext(const bsl::shared_ptr<NegotiationContext>& value);
 
     // ACCESSORS
 
     /// Return the value of the corresponding field.
-    bool                                   isIncoming() const;
-    void*                                  userData() const;
-    void*                                  resultState() const;
-    const bsl::shared_ptr<bmqio::Channel>& channel() const;
-    const InitialConnectionCompleteCb&     initialConnectionCompleteCb() const;
+    bool                                       isIncoming() const;
+    void*                                      userData() const;
+    void*                                      resultState() const;
+    const bsl::shared_ptr<bmqio::Channel>&     channel() const;
     const bsl::shared_ptr<NegotiationContext>& negotiationContext() const;
+
+    void complete(const int                               rc,
```

Comment:
```suggestion
    void complete(int                                     rc,
```

Basic arithmetic type